docs/guides/http/json-validation.md

```diff
@@ -0,0 +1,93 @@
+# JSON validation
+
+Bocadillo has built-in support for JSON data validation using [TypeSystem]. It allows you to perform schema-based validation, as well as use [orm] to get RESTful validation _à la_ Django REST Framework.
```

Comment:
Recommend

Bocadillo has built-in support for JSON data validation using [TypeSystem], a data validation library.

Reply:
Fixed 👍